README.md

```diff
@@ -1 +1,248 @@
+
+-Type git clone in the terminal, paste the URL you copied earlier, and press "enter" to create your local clone.
+
+```
+$ git clone https://github.com/AhmedMohsen600/shopping.git
+```
+
+# Install Dependencies(with following Command)
+
+- npm install
+
+# Build webpack <a name="setup"></a>
+
+- run command
+
+```
+npm run build
+```
+
+## Start server
+
+```
+npm start
+```
+
+### Prerequisites <a name="prerequisites"></a>
+
+In order to run this project you need:
+
+- [x] Basic Knowledge of **HTML**, **CSS**, & **JavaScript**.
+- [x] Basic knowledge of **ES6** syntax.
+- [x] Basic knowledge of **Git**, **GitHub** &**Gitflow**.
+- [x] Basic knowledge of **Linters**.
+- [x] Basic knowledge of **Webpack**.
+- [x] Basic knowledge of **Jest** unit-testing library.
+- [x] Basic knowledge of **VS Code** or any other code-editors.
+- [x] Basic knowledge of **React**.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+    _cd PATH_
+
+    _git clone https://github.com/AhmedMohsen600/shopping.git
+
+\_cd math-calc
+
+_npm install_
+
+_npm run build_
+
+_npm start_
+
+### Install
+
+Install this project with:
+
+```sh
+  npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  npm start
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  npm test
+```
+
+### Deployment <a name="triangular_flag_on_post-deployment"></a>
+
+You can deploy this project using:
+
+```sh
+  npm start
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Ahmed Mohsen**
+
+- GitHub: [@Ahmed Mohsen](https://github.com/AhmedMohsen600)
+- LinkedIn: [@Ahmed Mohsen](https://www.linkedin.com/in/ahmed-mohsen-bb75b2194/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Implement additional home pages, one page per category of movies**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Zainab-Alzubair/the-meal/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CREDITS -->
+
+## Credits <a name="credits"></a
+
+- The original design ideal by [Nelson Sakwa on Behance](https://www.behance.net/sakwadesignstudio)
+- Project from [Microverse](https://bit.ly/MicroverseTN) React module
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+Give a ⭐️ if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- Thanks to [Microverse](https://www.microverse.org/) for giving this opportunity to learn ...
+- Code Reviewers & Coding Partners.
+- Hat tip to anyone whose code was used.
```

Comment:
- [ ] [Optiona]There are small changes so I am giving it an optional. please gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) on your README Acknowledgements.